### PR TITLE
(Worklet) Add bubbles demo

### PR DESCRIPTION
**Pull request by AKAspanion**
This PR adds css-houdini-bubbles as a demo

Screenshot 2021-01-22 at 8 59 33 PM
Demo: https://codepen.io/spanion/pen/rNMgerx

### CODE REVIEW

**Review by nucliweb — APPROVED** [review]
😊Very nice,  I love it!

File: worklets/bubbles.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    "packageName": "css-houdini-bubbles",
+    "npmUrl": "https://www.npmjs.com/package/css-houdini-bubbles",
+    "workletUrl": "https://unpkg.com/css-houdini-bubbles/dist/bubbles.js",
+    "cdnUrl": "https://unpkg.com/css-houdini-bubbles/dist/bubbles.js",
+    "demoUrl": "https://codepen.io/spanion/full/rNMgerx",
+    "tags": ["paint"],
+    "customProps": {
+      "--bubbles-colors": {
+        "type": "string",
+        "default": "#007C8E, #7940c1"
```
Comment (una) [inline worklets/bubbles.json:12]: I'd recommend using readable colors for the demo, i.e. `deepskyblue, pink`

File: worklets/bubbles.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      },
+      "--bubbles-max-radius": {
+        "type": "number",
+        "default": "60",
+        "range": [30, 80],
+        "step": "2"
+      },
+      "--bubbles-total-num": {
+        "type": "number",
+        "default": "20",
```
Comment (una) [inline worklets/bubbles.json:28]: I like `10` or `12` as the default in this space -- it feels less crowded

**Review by una — CHANGES_REQUESTED** [review]
LOVE this effect! Just left a few very small comments for the demo 😄

File: worklets/bubbles.json (hunk 1 above)
Comment (AKAspanion) [reply worklets/bubbles.json]: Changed colors to `teal, blueviolet`.

File: worklets/bubbles.json (hunk 2 above)
Comment (AKAspanion) [reply worklets/bubbles.json]: Changed default value to `12`.

**Comment by AKAspanion** [comment]
Thanks for the suggestion @una. I have updated the same. 😀

**Comment by una** [comment]
🎉🎉🎉

**Pull request merged**